composer.json

```diff
@@ -23,15 +24,15 @@
         }
     },
     "require": {
-        "php": "^8.1",
+        "php": "^8.2",
         "spatie/url": "^2.1",
         "laravel/slack-notification-channel": "^2.4",
         "spatie/ssl-certificate": "^2.4",
-        "filament/filament": "^2.0"
+        "filament/filament": "^3.0"
     },
     "require-dev": {
         "phpunit/phpunit": "^9.5",
-        "orchestra/testbench": "^7.13",
+        "orchestra/testbench": "^8.5",
         "spatie/laravel-ray": "^1.31",
```

Comment:
We can remove this dependency 👁️ no longer needed.

Reply:
yep